src/app-layer-smtp.c

```diff
@@ -558,7 +560,7 @@ int SMTPProcessDataChunk(const uint8_t *chunk, uint32_t len,
  *
  * \param state The smtp state.
  *
- * \retval  0 On suceess.
+ * \retval  0 On succeess.
```

Comment:
Not sure if this improves things much :-P

Reply:
Yikes.